lib/collisions.js

```diff
@@ -4,7 +4,8 @@ function collisionDetection(impactor, asteroids){
   for(var i = 0; i < asteroids.length; i++ ) {
     var a = asteroids[i];
       if( (impactor.x + impactor.width/2) > (a.x - 20) && (impactor.x - impactor.width/2) < (a.x + 20) && (impactor.y + impactor.height/2) > (a.y - 20) && (impactor.y - impactor.height/2) < (a.y + 20) ) {
-        collisionDetected = true;
+        collisionDetected = a;
```

Comment:
Is it returning "a" to differentiate an asteroid/ship collision from an asteroid/bullet collision?